flipperdevices/Flipper-Android-App
# Search file manager

**Background**

This PR add functional search into new file manager sample

**Changes**

- Add search into file manager
- Fix internal visibility for ComposableSearchBar

**Test plan**

- Open some folder with sample
- Try press search button
- See search elements with cool animations
